src/tests/class_tests/openms/data/FIAMS_output/SerumTest_merged_10.mzML

```diff
@@ -0,0 +1,82 @@
+<?xml version="1.0" encoding="ISO-8859-1"?>
```

Comment:
you have accidentally added these mzML test files to your PR.
They should not be in the repo (even if you `git rm ...` them, they will still be there and increase the size of the git repository.
Solution: checkout the `develop` branch and create a new branch from there (which does not contain these mzML files). Then lookup how `git cherry-pick ...` works and only use the commits which do not contain the mzMLs (that only works if they are in a separate commit). Alternatively, manually safe your newly added files to a different folder, and copy them back into the new branch manually. Then add them to git version control as usual.